pkg/controller/elasticsearch/migration/migrate_data.go

```diff
@@ -8,8 +8,20 @@ import (
 	"context"
 	"strings"
 
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
```

Comment:
nit: new line in imports

Reply:
this is IntelliJ separating the various import repos automatically